### PR TITLE
Fix TypeScript error TS1046

### PR DESCRIPTION
**Pull request by tomtargosz**
# Summary
In the current version (1.1.1) running the TypeScript compiler gives an error 1046 for `CameraRoll.d.ts`:
image
This PR updates the type file to correctly follow the TypeScript definition file rules. The project now successfully compiles with no TypeScript errors.

## Test Plan

All that's needed for testing is to ensure running the TypeScript compiler doesn't result in any errors.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |

## Checklist

- [x] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [x] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: typings/CameraRoll.d.ts [hunk 1]
```diff
@@ -7,85 +7,81 @@
...
+     * Returns a Promise with photo identifier objects from the local camera
+     * roll of the device matching shape defined by `getPhotosReturnChecker`.
+     */
+    function getPhotos(params: GetPhotosParams): Promise<PhotoIdentifiersPage>;
 }
 
-let CameraRoll: CameraRollStatic;
-
-export default CameraRoll;
+export = CameraRoll;
```
Comment (bartolkaruza) [inline typings/CameraRoll.d.ts]: missing newline

Comment (tomtargosz) [reply typings/CameraRoll.d.ts]: Fixed 👍🏼

**Comment by tomtargosz** [comment]
@bartolkaruza Nope! TypeScript users shouldn't have to change anything in their code.

**Comment by tomtargosz** [comment]
@bartolkaruza is anything else needed for this to be merged in?

**Comment by bartolkaruza** [comment]
Thanks a lot! (took me a while to find the time to get around to these PR's, sorry)

**Pull request merged**